index.html

```diff
@@ -0,0 +1,2199 @@
+          </a>
+        </div> 
+        <div class="header_left col d-flex flex-lg-column align-items-lg-start ">
+          <div class="row align-items-center row justify-content-start">
+            <a href="#" class="d-none d-sm-block text-decoration-none">
+              <div class="d-flex justify-content-between">
+                <i class="fas fa-sun text-white"></i> 
+                  <span class="text-white"> 25°</span>
+                <span class="d-none d-md-inline-block text-white">Kampala District, UG ></span>
+              </div>
+            </a>
+          </div>
+          <div class="header-date row d-none d-lg-inline-block font-weight-normal pt-2">
+            Tue, Apr 13, 2020
+          </div>
+        </div>
+        <div class="col d-none d-lg-flex justify-content-center ">
+          <a href="#">
+            <img class="newsweek-logo" src="./newweek-logo.PNG" alt="newsweek">
+          </a>
+        </div>   
+        <div class=" d-flex align-items-center">
+          <div>
+            <a class="text-white font-weight-bold pr-3 text-decoration-none" href="#">SIGN IN</a>
+          </div>
+          <div>
+              <button type="button" class="header-subscribe_btn text-decoration-none">SUBSCRIBE ></button>
+          </div>
+          <span class="d-block d-lg-none menu-icon d-flex justify-content-center align-items-center">
+            <i class="fas fa-bars"></i>
+          </span>
+        </div>   
+      </div>
+    </header>
+
+    <nav class=" nav-bar navbar-expand-lg navbar-light navigation border-bottom">
+      <div class=" collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="d-flex flex-row align-items-center nav-list font-weight-bold text-body justify-content-between">
+          <li class="nav-item"><a class="border-right" href="#">U.S</a></li>
+          <li  class="nav-item"><a class="border-right" href="#">World</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Business</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Tech & Science</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Culture</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Newsgeek</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Sports</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Health</a></li>
+          <li class="nav-item"><a class="border-right" href="#">The Debate</a></li>
+          <li class="nav-item"><a class="border-right" href="#">Vantage</a></li>
+          <li class="nav-item"><a  href="#">Weather</a></li>
+          <li  class="nav-item nav-icons noPadding text-body d-flex flex-row">
+            <input class=" form-control border-0 search_input w-100" type="search" placeholder="Search" aria-label="Search">
+              <i class="fa fa-search text-body search-icon " aria-hidden="true"></i>
+          </li>
```

Comment:
Your nav elements should occupy 100% width of the navbar space. Please adjust them.
Yours:
![Screenshot from 2020-04-22 12-59-37](https://user-images.githubusercontent.com/51926937/80017036-8c7bae80-8499-11ea-8361-6d897edbf1c7.png)

Original:
![Screenshot from 2020-04-22 12-59-43](https://user-images.githubusercontent.com/51926937/80017047-91d8f900-8499-11ea-9b27-25a76b58b9cf.png)